udata/core/dataset/models.py

```diff
@@ -261,12 +261,18 @@ def check_availability(self):
 
         Return a list of booleans.
         """
+        remote_resources = [resource
+                            for resource in self.resources
+                            if resource.filetype == 'remote']
         # First, we try to retrieve all data from the group (slug).
-        error, response = check_url_from_group(self.slug)
+        if remote_resources:  # Avoid checking 404s.
+            error, response = check_url_from_group(self.slug)
+        else:
+            return []
         if error:
             # The group is unknown, the check will be performed by resource.
```

Comment:
Shouldn't you create the group if it doesn't exists so on the next check it will be created/cached ?